### PR TITLE
'Readme.md | technologies logos added'

### PR DESCRIPTION
**Pull request by francosauchelli**
Hi, I've added the technologies logo to the credits section in the Readme.md.

I hope it could be helpful.

Regards.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,10 +80,14 @@ You can [download](https://github.com/IIvexII/LGU-TimetableAPI/releases) the lat
...
 This software uses the following open source packages:
 
-- [Node.js](https://nodejs.org/)
-- [Express.js](https://expressjs.com/)
-- [Shield.io](https://shields.io/)
-- [JSDOM](https://github.com/jsdom/jsdom)
+- [<img src='https://www.vectorlogo.zone/logos/nodejs/nodejs-ar21.svg' width=64 align='center' >](https://nodejs.org/)
+- [<img src='https://www.vectorlogo.zone/logos/expressjs/expressjs-ar21.svg' width=64 align='center' >](https://expressjs.com/)
+- [<img src='https://www.vectorlogo.zone/logos/shieldsio/shieldsio-official.svg' width=64 align='center' >](https://shields.io/)
+- [<svg xmlns="http://www.w3.org/2000/svg" viewBox="-149 420 100 100" width=32 height=48 >
```
Comment (IIvexII) [inline README.md:86]: Please use the image instead of svg. And also remove the - to make them align horizontally.

**Review by IIvexII — CHANGES_REQUESTED** [review]
I reviewed your purposed changes. You have done a great job but can you please have a look at the comment  that I have made. Thanks!

**Comment by francosauchelli** [comment]
I've replaced svg with img and removed the "-" to align the logos horizontally.

Please any comments let me know.

File: README.md [hunk 2]
```diff
@@ -80,10 +80,11 @@ You can [download](https://github.com/IIvexII/LGU-TimetableAPI/releases) the lat
```
Comment (IIvexII) [inline README.md:80]: image

Can you please also change the text as seen in the screenshot and Credits to Technologies.

**Comment by IIvexII** [comment]
The list looks good. But I think you forget to change the text and can you also change the heading **Credit** to **Technologies**

File: README.md (hunk 2 above)
Comment (francosauchelli) [reply README.md:80]: No problem! I've just done it.

**Review by IIvexII — APPROVED** [review]
Thanks for contributing @francosauchelli. I appreciate your efforts ❤️.

**Pull request merged**